composer.json

```diff
@@ -11,7 +11,8 @@
   },
   "require-dev": {
     "composer/xdebug-handler": "^2.0",
-    "open-telemetry/dev-tools": "dev-main"
+    "open-telemetry/dev-tools": "dev-main",
+    "phpunit/phpunit": "^11.4"
```

Comment:
this might be too restrictive, since we support back to 8.0/8.1

Reply:
Why is it even necessary, `dev-tools` should include it all, no?

Reply:
I thought so too